src/components/Header.js

```diff
@@ -1,35 +1,24 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
 import './styles/header.css';
+import { Link } from 'react-router-dom';
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossOrigin="anonymous" referrerPolicy="no-referrer" />;
 
 function Header() {
   return (
-    <nav>
-      <div className="navbar links flex">
-        <ul className="flex">
-          <li>
-            <h2>
-              <a href="/books" className="logo">Bookstore CMS</a>
-
-            </h2>
-          </li>
-          <li>
-            <NavLink to="/">
-              Books
-            </NavLink>
-          </li>
-          <li>
-            <NavLink to="/categories">Categories</NavLink>
-
-          </li>
-
+    <nav className="panel-bg">
```

Comment:
- [ ] Good job making some changes. Kindly ensure that your header width fits the entire width of the viewport by giving it a width of 100% so that it match the Zeplin design 👍🏽 

your design 👇🏽 
![image](https://github.com/naji-97/sec-book-store/assets/63473584/988bfcb9-bd03-4b28-b4ea-a5d0d60846e1)

Zeplin design 👇🏽 
![image](https://github.com/naji-97/sec-book-store/assets/63473584/c2b16775-e866-4a1c-9a3e-e785029f6c64)